Description: Use system mocha
 use system mocha instead of npm installed mocha
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 REPORTER = dot
 
 test:
-	@./node_modules/.bin/mocha \
+	mocha \
 		--require test/common \
 		--reporter $(REPORTER) \
 		--growl \
